Description: Fix documentation files to meet debian policy
Author: Mahyuddin Susanto <udienz@gmail.com>

diff -Nur -x '*.orig' -x '*~' gkamus-0.3/Makefile.am gkamus-0.3.new/Makefile.am
--- gkamus-0.3/Makefile.am	2009-02-20 14:36:46.000000000 +0700
+++ gkamus-0.3.new/Makefile.am	2011-01-22 02:11:02.358966082 +0700
@@ -2,7 +2,7 @@
 
 SUBDIRS = src pixmaps data
 
-gkamusdocdir = ${prefix}/doc/gkamus
+gkamusdocdir = ${prefix}/share/doc/gkamus
 gkamusdoc_DATA = \
 	README\
 	COPYING\
diff -Nur -x '*.orig' -x '*~' gkamus-0.3/configure.in gkamus-0.3.new/configure.in
--- gkamus-0.3/configure.in	2009-02-20 14:36:46.000000000 +0700
+++ gkamus-0.3.new/configure.in	2011-01-22 02:10:42.479088547 +0700
@@ -33,7 +33,7 @@
 dnl Set PACKAGE DATA & DOC DIR
 dnl ===============================================================
 packagedatadir=share
-packagedocdir=doc/${PACKAGE}
+packagedocdir=share/doc/${PACKAGE}
 
 dnl ===============================================================
 dnl Set PACKAGE DIRS in config.h.
