Description: Fix desktop files to meet freedesktop standard policy
Author: Mahyuddin Susanto <udienz@ubuntu.com>

diff -Nur -x '*.orig' -x '*~' gkamus-0.3/gkamus.desktop.in gkamus-0.3.new/gkamus.desktop.in
--- gkamus-0.3/gkamus.desktop.in	2009-02-20 14:36:46.000000000 +0700
+++ gkamus-0.3.new/gkamus.desktop.in	2011-01-22 02:18:25.743126910 +0700
@@ -1,11 +1,12 @@
 [Desktop Entry]
 Version=1.0
 Name=gKamus
-Comment=Kamus Bahasa Inggris - Indonesia
-Exec=@prefix@/bin/gkamus
-Icon=@prefix@/share/pixmaps/gkamus.png
+Comment=English - Indonesia Dictionary
+Comment[id]=Kamus Bahasa Inggris - Indonesia
+Exec=gkamus
+Icon=gkamus
 Terminal=false
 Type=Application
-Categories=Application;Utility
+Categories=Education;Language;
 StartupNotify=true
 Terminal=false
diff -Nur -x '*.orig' -x '*~' gkamus-0.3/gkamus.spec gkamus-0.3.new/gkamus.spec
--- gkamus-0.3/gkamus.spec	2009-02-22 02:56:39.000000000 +0700
+++ gkamus-0.3.new/gkamus.spec	2011-01-22 02:18:30.778496484 +0700
@@ -3,8 +3,9 @@
 Name: gkamus
 Version: %{version}
 Release: 1
-Summary: Kamus Bahasa Inggris - Indonesia
-Group: Applications/Utility
+Comment[id]: Kamus Bahasa Inggris - Indonesia
+Comment: Simply dictionary English - Indonesia
+Group: Education/Language
 License: GPLv2
 URL: http://gkamus.sourceforge.net
 Vendor: http://gkamus.sourceforge.net
